userguide.md

```diff
@@ -2,8 +2,7 @@
 
 #### PL/R User’s Guide - R Procedural Language
 
-#### Copyright © 2003-2018 Joseph E Conway
-
+#### Copyright © 2003-2021 Joseph E Conway
```

Comment:
I'd just as soon remove the end date, it's of no use and requires us to update every year.

Reply:
* yes,  "Copyright © 2003 Joseph E Conway" can go there